src/components/flashcard/Flashcard.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import "./Flashcard.css";
+
+const FlashCard = ({ question, answer }) => {
```

Comment:
Since `Flashcard` is one word, we may not need to use `CamelCase` in naming the component.

Reply:
Got it! Thanks!

Reply:
Updated as per your request.